cpu/cc26x2_cc13x2/include/cc26x2_cc13x2_prcm.h

```diff
@@ -23,66 +23,186 @@
-#define DDI_0_OSC_CTL0_RCOSC_LF_TRIMMED             0x00001000
-#define DDI_0_OSC_CTL0_HPOSC_MODE_EN                0x00004000
-#define DDI_0_OSC_CTL0_DCDC_SRC_SEL                 0x01000000
-#define DDI_0_OSC_CTL0_DOUBLER_RESET_DURATION       0x02000000
-#define DDI_0_OSC_CTL0_DOUBLER_START_DURATION       0x0C000000
-#define DDI_0_OSC_CTL0_BYPASS_RCOSC_LF_CLK_QUAL     0x10000000
-#define DDI_0_OSC_CTL0_BYPASS_XOSC_LF_CLK_QUAL      0x20000000
-#define DDI_0_OSC_CTL0_XTAL_IS_24M                  0x80000000
+#define DDI_0_OSC_CTL0_SCLK_HF_SRC_SEL_m        0x00000001
+#define DDI_0_OSC_CTL0_SCLK_HF_SRC_SEL_s        0
+#define DDI_0_OSC_CTL0_SCLK_HF_SRC_SEL_XOSC     0x00000001
+#define DDI_0_OSC_CTL0_SCLK_HF_SRC_SEL_RCOSC    0x00000000
+#define DDI_0_OSC_CTL0_SCLK_LF_SRC_SEL_m        0x0000000C
+#define DDI_0_OSC_CTL0_SCLK_LF_SRC_SEL_s        2
+#define DDI_0_OSC_CTL0_ACLK_TDC_SRC_SEL         0x00000180
+#define DDI_0_OSC_CTL0_CLK_LOSS_EN              0x00000200
+#define DDI_0_OSC_CTL0_XOSC_LF_DIG_BYPASS       0x00000400
+#define DDI_0_OSC_CTL0_XOSC_HF_POWER_MODE       0x00000800
+#define DDI_0_OSC_CTL0_RCOSC_LF_TRIMMED         0x00001000
+#define DDI_0_OSC_CTL0_HPOSC_MODE_EN            0x00004000
+#define DDI_0_OSC_CTL0_CLK_DCDC_SRC_SEL_m       0x01000000
+#define DDI_0_OSC_CTL0_DOUBLER_RESET_DURATION   0x02000000
+#define DDI_0_OSC_CTL0_DOUBLER_START_DURATION   0x0C000000
+#define DDI_0_OSC_CTL0_BYPASS_RCOSC_LF_CLK_QUAL 0x10000000
+#define DDI_0_OSC_CTL0_BYPASS_XOSC_LF_CLK_QUAL  0x20000000
+#define DDI_0_OSC_CTL0_XTAL_IS_24M              0x80000000
+#define DDI_0_OSC_STAT0_PENDINGSCLKHFSWITCHING  0x00000001
+#define DDI_0_OSC_STAT0_SCLK_HF_SRC_m           0x10000000
+#define DDI_0_OSC_STAT0_SCLK_HF_SRC_s           28
+#define DDI_0_OSC_STAT0_SCLK_LF_SRC_m           0x60000000
+#define DDI_0_OSC_STAT0_SCLK_LF_SRC_s           29
 /** @} */
 
-/** @ingroup cpu_cc26x2_cc13x2_peripheral_memory_map
-  * @{
-  */
-#define DDI0_OSC_BASE       0x400CA000 /**< DDI0_OSC base address */
-/*@}*/
+/**
+ * @ingroup cpu_cc26x2_cc13x2_peripheral_memory_map
+ * @{
+ */
+#define DDI_DIR              0x00000000
+#define DDI_SET              0x00000080
+#define DDI_CLR              0x00000100
+#define DDI_MASK4B           0x00000200
+#define DDI_MASK8B           0x00000300
+#define DDI_MASK16B          0x00000400
+/**
+ * @brief   DDI0_OSC base address
+ */
+#define DDI0_OSC_BASE        (PERIPH_BASE + 0xCA000)
+/**
+ * @brief   DDI0_OSC 16-bit masked access base address
+ */
+#define DDI0_OSC_BASE_M16    (DDI0_OSC_BASE + DDI_MASK16B)
+/** @} */
+
+/**
+ * @brief   DDI_0_OSC register bank
+ */
+#define DDI_0_OSC            ((ddi0_osc_regs_t *) (DDI0_OSC_BASE))
+/**
+ * @brief   DDI_0_OSC 16-bit masked access register bank
+ */
+#define DDI_0_OSC_M16        ((ddi0_osc_regs_m16_t *) (DDI0_OSC_BASE_M16))
+
+/**
+ * @brief   Source clocks
+ * @{
+ */
+#define OSC_SRC_CLK_HF       0x00000001
+#define OSC_SRC_CLK_LF       0x00000004
```

Comment:
A comment about what these are doesn't hurt :wink:

Reply:
Updated the documentation :+1: